Online-Store/Online-Store/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿//using System;
+using System;
+
+namespace Online_Store
+{
+    internal class Program
+    {
+        static void Main(string[] args)
```

Comment:
Expected the folder separation, as well as service/model separation, as we discussed in class